bin/zopen-importenvs

```diff
@@ -37,19 +38,18 @@ then
 fi
   echo "final depsPath = $depsPath"
```

Comment:
This seems like debug output, should it be removed?

Reply:
yes, I have removed it.